app/controllers/metadata_controller.rb

```diff
@@ -45,13 +45,13 @@ def default_manifest
       name: SiteSetting.title,
       short_name: SiteSetting.short_title.presence || SiteSetting.title.truncate(12, separator: ' ', omission: ''),
       display: display,
-      start_url: Discourse.base_uri.present? ? "#{Discourse.base_uri}/" : '.',
+      start_url: Discourse.base_path.present? ? "#{Discourse.base_path}/" : '.',
```

Comment:
Do want `start_url` to be a relative URL? I see it supports those (https://developer.mozilla.org/en-US/docs/Web/Manifest/start_url), but since "the manifest URL is used as the base URL to resolve it" I'm not 100% sure the final URL is correct. Doesn't the "manifest URL" already contain the base_path? 🤔

Reply:
This isn't a functional change, base_uri and base_path are aliases

Reply:
For a subfolder, it will never be a **relative** URL. base_path includes a `/` at the beginning for subfolder sites, so I think it will be an **absolute** URL.

Although, reading that, it sounds like we could probably get away with `'.'` even for subfolder sites... But I am inclined not to change it - the current implementation of start_url is working fine. The diff here is just changing `base_uri` for the less confusing `base_path`

Reply:
I understand that. 🙂  My questions are: was it correct in the first place, and if it was, isn't it better to be explicit and use `Discourse.base_url`? Even just for the sake of people looking at the code and thinking "a path in a *_url field?" 😉

Reply:
Oops, it didn't load your comment for me, @davidtaylorhq. 👀